pyaedt/modeler/circuits/PrimitivesEmit.py

```diff
@@ -743,6 +743,99 @@ def band_stop_frequency(self, band_node, units=""):
             units = self.units["Frequency"]
         return consts.unit_converter(float(band_node.props["StopFrequency"]), "Freq", "Hz", units)
 
+    def set_band_start_frequency(self, band_node, band_start_freq, units=""):
+        """Set start frequency of a given band.
+
+        Parameters
+        ----------
+        band_node
```

Comment:
If possible, please add the type of this parameter. I am guessing it is not a built-in type.

Reply:
Done.